apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -276,7 +277,8 @@ function getParamsForAssessment(
         ? assessment.type === 'Exam'
         : assessment.shuffleQuestions,
     allow_issue_reporting: assessment.allowIssueReporting,
-    allow_real_time_grading: assessment.allowRealTimeGrading,
+    allow_real_time_grading: assessment.allowRealTimeGrading ?? true,
```

Comment:
nit: Can you add a small comment explaining that `allow_real_time_grading` is the 'resolved' setting (e.g. with a default)? Additionally, is there a reason why we can't switch the code over to using `json_allow_real_time_grading` now, and after it's read from the database perform the `?? true`.